apis/python/test/test_index.py

```diff
@@ -9,8 +9,9 @@
 def test_flat_index(tmp_path):
     uri = os.path.join(tmp_path, "array")
     index = flat_index.create(uri=uri, dimensions=3, vector_type=np.dtype(np.uint8))
-    result_d, result_i = index.query(np.array([[2, 2, 2]], dtype=np.float32), k=3)
-    assert {ind.MAX_UINT64} == set(result_i[0])
+    for _ in range(2):
+        result_d, result_i = index.query(np.array([[2, 2, 2]], dtype=np.float32), k=3)
+        assert {ind.MAX_UINT64} == set(result_i[0])
```

Comment:
Lets add a util function that runs x queries and asserts results to avoid duplication and make the code more readable.